### PR TITLE
Add shelley cli commands to create a multisig script & convert a script to an address

### PR DESCRIPTION
**Pull request by Jimbo4350**

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/Shelley/Run/Address.hs [hunk 1]
```diff
@@ -181,3 +182,21 @@ readAddressVerificationKeyFile (VerificationKeyFile vkfile) =
...
+  -> NetworkId
+  -> Maybe OutputFile
+  -> ExceptT ShelleyAddressCmdError IO ()
+runAddressBuildScript (ScriptFile fp) nId mOutFp = do
+  script <- firstExceptT ShelleyAddressCmdReadFileError
+              . newExceptT $ readFileTextEnvelope AsScript fp
+  let payCred = PaymentCredentialByScript $ scriptHash script
+      scriptAddr = serialiseAddress $ makeShelleyAddress nId payCred NoStakeAddress
+  case mOutFp of
+    Just (OutputFile oFp) -> liftIO $ Text.writeFile oFp scriptAddr
```
Comment (Jimbo4350) [inline cardano-cli/src/Cardano/CLI/Shelley/Run/Address.hs:201]: @duncan Are we purposely not writing the `Address Shelley` in the `TextEnvelope` format?

Comment (dcoutts) [reply cardano-cli/src/Cardano/CLI/Shelley/Run/Address.hs:201]: Yes. Addresses have their own standard presentation format.

File: cardano-cli/src/Cardano/CLI/Shelley/Run/Address.hs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
```
Comment (intricate) [inline cardano-cli/src/Cardano/CLI/Shelley/Run/Address.hs:2]: Might be able to remove this:

```suggestion
```

**Comment by mark-stopka** [comment]
This can be now merged as #1660 has been merged.

**Pull request merged**